packages/rpc-methods/CHANGELOG.md

```diff
@@ -6,11 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.38.0-flask.1]
+### Added
+- Add `snap_getLocale` ([#1557](https://github.com/MetaMask/snaps/pull/1557))
```

Comment:
```suggestion
- Add `snap_getLocale` JSON-RPC method ([#1557](https://github.com/MetaMask/snaps/pull/1557))
   - This will let snaps get the user locale from the extension.
```